GoInfoGame/GoInfoGameTests/quests/TestQuestUtils.swift

```diff
@@ -13,6 +13,14 @@ class TestQuestUtils {
         return Int64(Date().timeIntervalSince1970)
     }
     
+    static func olderThan(years: Int) -> Int64 {
```

Comment:
rename to `addTime` and enhance the function to take months, days and years as input. Just years will not work out

Reply:
resolved